### PR TITLE
Make collections official via menu

### PR DESCRIPTION
**Pull request by ranquild**
Epic https://github.com/metabase/metabase/issues/22826

How to test:
- Run Metabase EE
- Go to a collection that is not root or personal
- Make sure it is possible to make the collection official/regular via the header menu

Please note that `Edit this collection` menu option will be removed in a follow-up PR

Screenshot 2022-06-24 at 17 38 01

### CODE REVIEW

**Review by alxnddr — COMMENTED** [review]
It shows the option for the Our Analycis collection and it does not change anything, probably it should not

File: enterprise/frontend/src/metabase-enterprise/collections/index.js [hunk 1]
```diff
@@ -19,7 +19,33 @@ PLUGIN_COLLECTIONS.REGULAR_COLLECTION = REGULAR_COLLECTION;
...
+        action: () =>
+          onUpdate(collection, {
+            authority_level: OFFICIAL_COLLECTION.type,
+          }),
+      },
+    ];
+  } else {
+    return [
+      {
+        title: "Make collection unofficial",
```
Comment (alxnddr) [inline enterprise/frontend/src/metabase-enterprise/collections/index.js:37]: Missing l10n here too

Comment (ranquild) [reply enterprise/frontend/src/metabase-enterprise/collections/index.js]: Fixed

File: enterprise/frontend/src/metabase-enterprise/collections/index.js [hunk 2]
```diff
@@ -19,7 +19,33 @@ PLUGIN_COLLECTIONS.REGULAR_COLLECTION = REGULAR_COLLECTION;
 
 PLUGIN_COLLECTIONS.AUTHORITY_LEVEL = AUTHORITY_LEVELS;
 
-PLUGIN_COLLECTIONS.authorityLevelFormFields = [
+PLUGIN_COLLECTIONS.getAuthorityLevelMenuItems = (collection, onUpdate) => {
+  if (isRegularCollection(collection)) {
+    return [
+      {
+        title: `Make collection official`,
```
[the remark this answers is not in the document]
Comment (ranquild) [reply enterprise/frontend/src/metabase-enterprise/collections/index.js]: Fixed

**Comment by ranquild** [comment]
> It shows the option for the Our Analycis collection and it does not change anything, probably it should not

Thanks, fixed

**Review by alxnddr — APPROVED** [review]

**Pull request merged**